arrays_strings/unique_chars/unique_chars_solution.ipynb

```diff
@@ -278,22 +278,23 @@
   }
  ],
  "metadata": {
+  "celltoolbar": "Edit Metadata",
```

Comment:
Yes it was an accident. It will great if you can ignore that.

I have to fix my jupyter notebook.